templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -568,6 +568,14 @@
 			{{end}}
 		{{end}}
 
+		<div class="ui divider"></div>
+		<div class="ui equal width compact grid">
+			<div class="row ac">
+				<span class="text column truncate">{{.i18n.Tr "repo.issues.reference_link" .Issue.ReferenceLink}}</span>
+				<button class="ui two wide button column p-3" data-clipboard-text="{{.Issue.ReferenceLink}}">{{svg "octicon-copy" 14}}</button>
```

Comment:
```suggestion
				{{$issueReferenceLink := printf "%s#%d" .Issue.Repo.FullName .Issue.Index}}
				<span class="text column truncate">{{.i18n.Tr "repo.issues.reference_link" $issueReferenceLink}}</span>
				<button class="ui two wide button column p-3" data-clipboard-text="{{$issueReferenceLink}}">{{svg "octicon-copy" 14}}</button>
```

This should be enough, then `issue.go ReferenceLink` could be removed because it's not used anywhere else.

Reply:
Thanks again! I've changed and tested the code locally, and it seems to work as intended. I've put it in commit [c3f7c4c](https://github.com/go-gitea/gitea/pull/19609/commits/c3f7c4c0467e4f29d447128a5983c14963955afa)